resources/js/Pages/Admin/Rooms/CreateRoomForm.vue

```diff
@@ -35,6 +36,36 @@
                         <jet-input-error :message="createRoomForm.error('building')" class="mt-2"/>
                     </div>
 
+                    <div class="col-span-6 sm:col-span-3">  
+                        <jet-label for="stand_capacity" value="Standing Capacity"/>
+                        <jet-input id="stand_capacity" type="stand_capacity" class="mt-1 block w-full" v-model="createRoomForm.capacity_standing"/>
+                        <jet-input-error :message="createRoomForm.error('stand_capacity')" class="mt-2"/>
```

Comment:
![image](https://user-images.githubusercontent.com/21128579/105226687-55bcec80-5b2e-11eb-9673-6916d7617ca1.png)
Some numerical front end validation is missing

Reply:
Front end now validates this.